go.mod

```diff
@@ -5,7 +5,7 @@ go 1.23.0
 require (
 	github.com/AlessandroPomponio/go-gibberish v0.0.0-20191004143433-a2d4156f0396
 	github.com/caarlos0/env/v9 v9.0.0
-	github.com/cilium/ebpf v0.16.0
+	github.com/cilium/ebpf v0.17.0
```

Comment:
This version had a Bug. v0.17.1 is encouraged: https://github.com/cilium/ebpf/releases/tag/v0.17.1